packages/nameguard-sdk/src/index.ts

```diff
@@ -499,22 +505,22 @@ class NameGuard {
    *
    * @param {string} name The name for NameGuard to inspect.
    * @param {InspectNameOptions} options The options for the inspection.
-   * @returns {Promise<NameGuardReport>} A promise that resolves with the `NameGuardReport` of the name.
+   * @returns {Promise<ConsolidatedNameGuardReport>} A promise that resolves with the `NameGuardReport` of the name.
```

Comment:
`NameGuardReport extends ConsolidatedNameGuardReport` should it be here `ConsolidatedNameGuardReport | NameGuardReport` or just `ConsolidatedNameGuardReport` is fine?

Reply:
I would go for the union since then IDEs can suggest fields from the child class.